Create 2024-05-10-sendgrid-transactional-emails.md (#8990)
pantheon-systems/documentation: source/releasenotes/2024-05-10-sendgrid-transactional-emails.md

---

> title: "Transactional Emails: Rest API emails now sent via SendGrid effective May 12, 2024"

Reviewer: ```suggestion
title: "Transactional Emails: Rest API emails now sent via SendGrid effective May 12, 2024"
```

This wording is confusing @tyherbert. Are the emails "REST API emails" or are they just emails from WordPress and Drupal that will now be delivered by SendGrid instead of the previous method (which I think simply relied on the capacity of any server running to a CMS to try to send an email) and the mechanism of communication between our PHP runtime and SendGrid is SendGrid's REST API?

Reviewer: I'm also not sure the word "transactional" is helping here. It might risk adding confusion. If we just mean "Emails sent by WordPress or Drupal" then let's say something plainer.

Author: I agree with you about the confusion. In this case, the term transactional email is and email type (generally, you see transactional vs marketing emails). They are categorized differently by email providers (marketing emails have different standards, opt-in, etc). If we call out transactional emails once (maybe in the title), we can remove this from the other areas.

Author: How does this look?
```suggestion
title: "Transactional Emails: Now sent via SendGrid effective May 12, 2024"
```

published_date: "2024-05-10"
categories: [new-feature] [WordPress] [Drupal]
---

Effective May 12th - We will begin rolling out SendGrid to send transactional emails.

> * Pantheon will begin using SendGrid to send customer transactional emails via rest API during the week of May 12th, 2024 (we expect the full cutover to be complete by the end of May).

Reviewer: ```suggestion
* Pantheon will begin using SendGrid to send customer transactional emails via REST API during the week of May 12th, 2024 (we expect the full cutover to be complete by the end of May).
```

REST should be capitalized. But is that a relevant detail for customers? Or do they just need to know that emails will now come from SendGrid for messages originating from WordPress or Drupal?

Author: How about this?
```suggestion
* Pantheon will begin using SendGrid to send during the week of May 12th, 2024 (we expect the full cutover to be complete by the end of May).
```

* You don't need to make any changes (Pantheon will manage the cutover).

> * All emails sent via SMTP forwarding will not be changed, and no impact should be expected.

Author: ```suggestion
* All emails sent via SMTP forwarding will not be changed, and no impact should be expected.
* Pantheon encourages customers to use third-party email providers, to better manage their email reputation.
```

---
title: "Transactional Emails now sent via SendGrid API"
published_date: "2024-05-10"
categories: [new-feature, wordpress, drupal]
---
Beginning this week, Pantheon will begin cutting over individual WordPress and Drupal sites to use Sendgrid's API for the delivery of outgoing email initiated by these CMSes. We expect all sites to be cut over by May 31st. This change is happening at the platform level, not CMS code or configuration, and therefore requires no action by customers. This change should not be noticeable by most customers aside from increased reliability of emails initiated by WordPress and Drupal.

WordPress and Drupal core can trigger transactional emails for behaviors like the "Forgot Password" flow that allows someone to reset their password after receiving an email.

![WordPress forgot password form](../images/wp-forgot-password-form.png)

> Historically Pantheon has relied on a local MTA (postfix) to send these messages and it has been common for such emails to be categorized as spam or otherwise fail. For teams for whom the delivery of such transactional emails is crtical to operation, we have recommended relying on third party services and continue to do so even as we make our platform-level emails more reliable with this switch to Sendgrid.

Reviewer: ```suggestion
Historically Pantheon has relied on a local MTA (postfix) to send these messages and it has been common for such emails to be categorized as spam or otherwise fail. For teams for whom the delivery of such transactional emails is critical to operation, we [have recommended relying on third-party services](email) and continue to do so even as we make our platform-level emails more reliable with this switch to Sendgrid.
```

Reviewer 2: I'll add a crosslink to the email doc next week sometime in a follow-up PR, since that doc is almost entirely stale following this rollout we are planning a full rewrite

---
title: "Transactional emails now sent via SendGrid API"
published_date: "2024-05-10"
categories: [new-feature, wordpress, drupal]
---
Beginning this week, Pantheon will start transitioning individual WordPress and Drupal sites to use Sendgrid's API for the delivery of outgoing email initiated by these CMSes. We expect all sites to be fully transitioned by May 31, 2024. This change is happening at the platform level, not within the CMS code or configuration, and therefore requires no action by customers. 

> <h3>Reason for transition</h3>

Reviewer: ```suggestion
## Reason for transition
```

We should stick with markdown syntax for headings in markdown files. I'm also switching it to an h2 since the title is the h1
